x-pack/plugins/cases/public/components/markdown_editor/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
I think we don't need this file.

Reply:
except this one, you're right. this is not used ;)